src/App.test.js

```diff
@@ -1,6 +1,7 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
+// comment
```

Comment:
I'd definitely keep this test up to date at a high level, maybe setting up something like [TravisCI](https://travis-ci.org/) to automatically run the test and make sure your master branch always at least renders the first page!